handbook/engineering/search/product.md

```diff
@@ -38,6 +38,16 @@ We work in two-week iterations.
 
 Iterations start every other Monday.
 
+#### Planning artifacts
+
+We use two main artifacts to plan iterations: our [planning document](https://docs.google.com/document/d/1swnkQwd724IB_HP3_Mw3KkFnfl45t-PmWcNo_1gtkQM/edit#), and a GitHub project specific to the iteration. We think of their relative roles as follows:
+
+- The planning document is used as a discussion platform to agree on intended outcomes for the iteration. An intended outcome is not "issue #xxxx is closed", but rather "problem X is solved".
+- The GitHub project is an itemized representation of the work needed to accomplish the outcomes listed in the planning document.
+- If some items in the GitHub project are not closed by the end of the iteration, teammates should mention in their last weekly update of the iteration what this impact is on planned outcomes.
```

Comment:
I tend to have a number of work items that are "things I'd like to get to, but are non-priority work". Should these be represented the same way as "work that I plan to complete to keep us on track for our goals"? 

If so, I imagine that, each iteration, there will be a handful of these items that aren't complete (I like to keep a deep wishlist).  

Should these items be part of planning at all? Or should they just be bonus work items?

Last iteration, I didn't put many of these items on the project to start, but added the PRs for them to the project for tracking purposes. It doesn't help with planning, but it at least recorded what was worked on after the fact

Reply:
We should plan with built-in slack time: the time you plan to allocate to roadmap goals is not 100% of your available engineering time. "things I'd like to get to, but are non-priority work" are tackled as part of this slack time.

On the other hand, the planning document + initial set of issues in the project board represents "work that I plan to complete to keep us on track for our goals".

I reckon adding them to the iteration project board when _completed_ would be a good way of surfacing what was accomplished. Depending on the nature of the task, mentioning it in weekly updates may be more appropriate -- user's discretion.

What do you think?

Reply:
Makes a lot of sense to me -- thanks for clarifying!

Reply:
I added a section about slack time based on what I wrote above to this process.